outputs/aws/s3/s3.go

```diff
@@ -0,0 +1,119 @@
+package s3
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	awssdk "github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+
+	aws "github.com/falco-talon/falco-talon/internal/aws/client"
+	"github.com/falco-talon/falco-talon/internal/events"
+	"github.com/falco-talon/falco-talon/internal/rules"
+	"github.com/falco-talon/falco-talon/outputs/model"
+	"github.com/falco-talon/falco-talon/utils"
+)
+
+type Config struct {
+	Bucket string `mapstructure:"bucket" validate:"required"`
+	Prefix string `mapstructure:"prefix" validate:"required"`
+}
+
+func Output(output *rules.Output, data *model.Data) (utils.LogLine, error) {
+	parameters := output.GetParameters()
+	var config Config
+	err := utils.DecodeParams(parameters, &config)
+	if err != nil {
+		return utils.LogLine{
+			Objects: nil,
+			Error:   err.Error(),
+			Status:  "failure",
+		}, err
+	}
+
+	config.Prefix = strings.TrimSuffix(config.Prefix, "/")
+	config.Prefix = strings.TrimPrefix(config.Prefix, "/") + "/"
+
+	key := fmt.Sprintf("%v_%v_%v_%v", time.Now().Format("2006-01-02T15-04-05Z"), data.Namespace, data.Pod, strings.ReplaceAll(data.Name, "/", "_"))
```

Comment:
Good naming convention, thinking in high scale though I'm wondering if we should add some kind of cluster identification on those files/